### DISCUSSION TITLE
[Telerik] More Telerik Detections (Report Viewer)

### DISCUSSION THREAD

**Discussion by nicpenning** [blacklanternsecurity/bbot#1439] [Module Requests]
The Telerik module today tries to identify telerik usage.

In light of recent Telerik vulnerabilities being announced,  it may be worthwhile to identify more ways to find Telerik.

With the POC exploit code, I found some URLs but need advice from the community to see if it's worthwhile to add these to the current module or if there is a better way to detect Telerik Report Viewer. 

[CVE-2024-4358/exploit.py at main · RevoltSecurities/CVE-2024-4358 (github.com)](https://github.com/RevoltSecurities/CVE-2024-4358/blob/main/exploit.py) Found these URLs

{url}/api/reports/clients 
{url}/api/reportserver/report 
{url}/api/reports/clients/{id}/parameters 
{url}/Token 
{url}/Startup/Register

The use case here is finding Telerik Report Viewer (and version if possible), not necessarily the vulnerabilities as I'm sure vulnerability scanning modules may do that.

**Comment by TheTechromancer** [comment]
@liquidsec should be interested to see this

**Comment by liquidsec** [comment]
Just a note that I am finally going to try to find time to add this to our telerik module soon

**Comment by liquidsec** [comment]
I'm going to go ahead and close this one... The reason is, is that the only reason we had a telerik module is those vulnerabilities were not easily solved with a nuclei module... 

these newer vulns actually have nothing to do with "telerik UI"  (separate telerik product) and are well covered by other tools.
